packages/features/ee/workflows/lib/reminders/reminderScheduler.ts

```diff
@@ -141,7 +141,7 @@ export const scheduleWorkflowReminders = async (args: ScheduleWorkflowRemindersA
     calendarEvent: evt,
     isNotConfirmed = false,
     isRescheduleEvent = false,
-    isFirstRecurringEvent = false,
+    isFirstRecurringEvent = true,
```

Comment:
@Udit-takkar  `isFirstRecurringEvent` should default to `true` if it is normal event and not recurring. Why? The first recurring event is handled the same as a normal event